### PR TITLE
Its almost over

### PR DESCRIPTION
**Pull request by muydanny**
User story 45 spec

45 BOOM

### CODE REVIEW

File: spec/features/merchant/items/index_spec.rb [hunk 1]
```diff
@@ -86,15 +86,54 @@
       expect(page).to_not have_css("#item-#{@chain.id}") 
   
     end
+    it "I see a link to add a new item" do
```
[the remark this answers is not in the document]
Comment (Gallup93) [reply spec/features/merchant/items/index_spec.rb:145]: Goo d!

File: app/views/merchant/items/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<h1>Create New Item For <%=link_to @merchant.name, "/merchants/#{@merchant.id}" %></h1>
+
+<center>
+  <%= form_tag "/merchant/items", method: :post do %>
```
Comment (stellakunzang) [inline app/views/merchant/items/new.html.erb:4]: superb

**Pull request merged**